examples/md-flexible/tests/domainDecomposition/RegularGridDecompositionTest.cpp

```diff
@@ -16,156 +16,183 @@
+  configuration.boundaryOption.value = {options::BoundaryTypeOption::periodic, options::BoundaryTypeOption::periodic,
+                                        options::BoundaryTypeOption::periodic};
+
+  auto domainDecomposition = std::make_shared<RegularGridDecomposition>(configuration);
+  const auto &localBoxMin = domainDecomposition->getLocalBoxMin();
+  const auto &localBoxMax = domainDecomposition->getLocalBoxMax();
+
+  auto autoPasContainer = std::make_shared<autopas::AutoPas<ParticleType>>();
+  autoPasContainer->setBoxMin(localBoxMin);
+  autoPasContainer->setBoxMax(localBoxMax);
   autoPasContainer->setCutoff(configuration.cutoff.value);
-  autoPasContainer->setRelativeOptimumRange(configuration.relativeOptimumRange.value);
-  autoPasContainer->setMaxTuningPhasesWithoutTest(configuration.maxTuningPhasesWithoutTest.value);
-  autoPasContainer->setRelativeBlacklistRange(configuration.relativeBlacklistRange.value);
-  autoPasContainer->setEvidenceFirstPrediction(configuration.evidenceFirstPrediction.value);
-  autoPasContainer->setExtrapolationMethodOption(configuration.extrapolationMethodOption.value);
-  autoPasContainer->setNumSamples(configuration.tuningSamples.value);
-  autoPasContainer->setMaxEvidence(configuration.tuningMaxEvidence.value);
-  autoPasContainer->setSelectorStrategy(configuration.selectorStrategy.value);
-  autoPasContainer->setTuningInterval(configuration.tuningInterval.value);
-  autoPasContainer->setTuningStrategyOption(configuration.tuningStrategyOption.value);
-  autoPasContainer->setMPIStrategy(configuration.mpiStrategyOption.value);
-  autoPasContainer->setVerletClusterSize(configuration.verletClusterSize.value);
-  autoPasContainer->setVerletRebuildFrequency(configuration.verletRebuildFrequency.value);
   autoPasContainer->setVerletSkin(configuration.verletSkinRadius.value);
-  autoPasContainer->setAcquisitionFunction(configuration.acquisitionFunctionOption.value);
+  autoPasContainer->setAllowedContainers({autopas::ContainerOption::directSum});
+  autoPasContainer->setAllowedTraversals({autopas::TraversalOption::ds_sequential});
   autoPasContainer->init();
-}
-}  // namespace
 
-TEST_F(RegularGridDecompositionTest, testGetLocalDomain) {
-  std::array<double, 3> globalBoxMin = {1.0, 1.0, 1.0};
-  std::array<double, 3> globalBoxMax = {10.0, 10.0, 10.0};
-  std::array<bool, 3> subdivideDimension = {true, true, true};
+  return std::make_tuple(autoPasContainer, domainDecomposition);
+}
 
-  RegularGridDecomposition domainDecomposition(
-      globalBoxMin, globalBoxMax, subdivideDimension, 0, 0,
-      {options::BoundaryTypeOption::periodic, options::BoundaryTypeOption::periodic,
-       options::BoundaryTypeOption::periodic});
+/**
+ * Setup 27 particle positions INSIDE the given container.
+ *
+ * Particles are placed in each corner (8), in the middle of every edge (12), and face (6),
+ * as well as the center of the domain. The positions are cutoff/2 away from the borders.
```

Comment:
```suggestion
 * Particles are placed near each corner (8), the middle of each edge (12), and the center of each face (6) such that   
 * they are cutoff/2 away from the nearby borders. In addition, a further particle is placed in the center of the domain.
```
Not entirely sure if this is the best rewrite though